badgecheck/reducers/graph.py

```diff
@@ -1,16 +1,45 @@
+import copy
+
 from ..actions.action_types import ADD_NODE, UPDATE_NODE
 
+current_node_id = -1
```

Comment:
Agree. Changing to `current_blank_node_number` to make it clear that this number is not an id itself, it's just a number that helps us assign blank ids as needed.